kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -221,3 +217,19 @@ private fun DeclarationDescriptor.extractKDocString(): String? {
             ?: extractSerializedKdocString()
 }
 
+/**
+ * Parse kdoc removing surrounding stuff including lead and tail blank lines.
+ * Remove each line indent with leading asterisk.
+ * @return non-empty list of strings or null
+ */
+private fun DeclarationDescriptor.extractKDocLines(): List<String>? {
```

Comment:
Do we really need to parse the comments to fix indents?
I mean, why is it not enough to split a comment into lines, trim starts and then join lines back?

Reply:
>  Do you suggest to keep the leading asterisk in place?

I asked if we could avoid attempting to parse the comment, and just mess with whitespaces and `\n` inside it instead.

> Essentially, would it be simpler or provide better alignment than now?

It might be safer and/or simpler.

Alternatively, we could rely on the existing parser. See `KDocSection.getContent()`.